addons/marker_flags/CfgVehicles.hpp

```diff
@@ -0,0 +1,55 @@
+class CfgVehicles {
+    class Man;
+    class CAManBase: Man {
+        class ACE_SelfActions {
+            class ACE_Equipment {
+                class ADDON {
+                    displayName = CSTRING(actionPlace);
+                    condition = QUOTE(_player call FUNC(canPlace));
+                    insertChildren = QUOTE(_this call FUNC(addActions));
+                    statement = "";
+                    exceptions[] = {};
+                    showDisabled = 0;
+                    priority = 1;
+                };
+            };
+        };
+    };
+
+    class FlagMarker_01_F;
+    class GVAR(white): FlagMarker_01_F {
+        scope = 2;
+        scopeCurator = 2;
+        author = ECSTRING(common,ACETeam);
+        displayName = CSTRING(white);
+        hiddenselectionstextures[] = {"#(argb,8,8,3)color(1,1,1,1.0,CO)"};
```

Comment:
`1.0` to just `1` and lowercase `co`. Also, proper capitalization for config entry (`hiddenSelectionsTextures`).